src/irmin-pack/unix/stats_intf.ml

```diff
@@ -82,6 +84,109 @@ module File_manager = struct
   [@@deriving irmin]
 end
 
+module Latest_gc = struct
```

Comment:
I think this naming can be improved. Could we name this `Gc` and then rename `t` to be `latest`? I think that `stats` is the `t` for this module.